Scripts/Features/ImportRegistryFile.ps1

```
@@ -21,6 +21,14 @@ function ImportRegistryFile {
     $importScript = {
         param($targetRegFilePath, $hiveContext)
 
+        $isWhatIf = $null -ne $script:Params -and $script:Params.ContainsKey("WhatIf")
+        if ($isWhatIf) {
+            Write-Host "[WhatIf] Previewing registry changes for $path:" -ForegroundColor Cyan
```

PSScriptAnalyzer (1.25.0)

[error] Variable reference is not valid. ':' was not followed by a valid variable name character. Consider using ${} to delimit the name.

(InvalidVariableReferenceWithDrive)